### PR TITLE
Updates type definitions for Dropzone 5.0.0

### PR DESCRIPTION
**Pull request by tedbcsgpro**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<http://www.dropzonejs.com/#news>>
- [x] Increase the version number in the header if appropriate.

@nvivo @a904guy

Rationale for updates:
* Dropzone.js 5.0.0 recently released
* Added strongly typed support for new client-side image resizing functionality in 5.0.0
* Prior type definition did not expose some methods our team finds useful

### CODE REVIEW

File: types/dropzone/dropzone-tests.ts [hunk 1]
```diff
@@ -146,6 +154,8 @@ const dropzone = new Dropzone(".test");
 dropzone.enable();
 dropzone.disable();
 
+
```
Comment (benbayard) [inline types/dropzone/dropzone-tests.ts]: two new lines?

File: types/dropzone/index.d.ts [hunk 2]
```diff
@@ -41,6 +39,12 @@ declare namespace Dropzone {
 		maxThumbnailFilesize?: number;
 		thumbnailWidth?: number;
 		thumbnailHeight?: number;
+		thumbnailMethod?: string;
+		resizeWidth?: number;
+      	resizeHeight?: number;
```
Comment (benbayard) [inline types/dropzone/index.d.ts]: indentation differences

File: types/dropzone/index.d.ts [hunk 3]
```diff
@@ -148,18 +156,30 @@ declare class Dropzone {
...
 
+	resizeImage(file:Dropzone.DropzoneFile, width?: number, height?: number, resizeMethod?: string, callback?:(...args:any[]) => void):void;
+
 	processQueue():void;
 
 	cancelUpload(file:Dropzone.DropzoneFile):void;
 
+	createThumbnail(file:Dropzone.DropzoneFile, width?:number, height?:number, resizeMethod?:string, fixOrientation?: boolean, callback?:(...args:any[]) => void):any;
+
+	createThumbnailFromUrl(file:Dropzone.DropzoneFile, width?: number, height?: number, resizeMethod?:string, fixOrientation?: boolean, callback?:(...args:any[]) => void, crossOrigin?:string):any;
```
Comment (benbayard) [inline types/dropzone/index.d.ts]: space after colon

File: types/dropzone/index.d.ts [hunk 4]
```diff
@@ -148,18 +156,30 @@ declare class Dropzone {
...
 
 	processFile(file:Dropzone.DropzoneFile):void;
 
 	uploadFile(file:Dropzone.DropzoneFile):void;
 
+	uploadFiles(files:Dropzone.DropzoneFile[]):void;
+
 	getAcceptedFiles():Dropzone.DropzoneFile[];
 
+	getActiveFiles(): Dropzone.DropzoneFile[]
```
Comment (benbayard) [inline types/dropzone/index.d.ts]: semicolon

File: types/dropzone/dropzone-tests.ts [hunk 5]
```diff
@@ -14,8 +18,14 @@ const dropzoneWithOptions = new Dropzone(".test", {
 	paramName: "file",
 	createImageThumbnails: true,
 	maxThumbnailFilesize: 1024,
-	thumbnailWidth: 50,
-	thumbnailHeight: 50,
+	thumbnailWidth: 120,
+	thumbnailHeight: 120,
+	thumbnailMethod: 'crop',
+	resizeWidth: 1024,
+    resizeHeight: 1024,
```
Comment (benbayard) [inline types/dropzone/dropzone-tests.ts]: indentation differences

**Comment by tedbcsgpro** [comment]
@benbayard Updated per your comments.

**Review by rbuckton — CHANGES_REQUESTED** [review]
As this is a major version update for the package, please follow the guidelines in the readme regarding maintaining historical versions: https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version

**Comment by tedbcsgpro** [comment]
@rbuckton done

**Review by rbuckton — APPROVED** [review]

**Comment by rbuckton** [comment]
Thanks for the contribution!

**Pull request merged**